src/ai/ai.service.ts

```
@@ -35,10 +31,11 @@ Line [Y]: [corrected code]
 Code changes to review:
 ${diff}`;
 
-      const result = await model.generateContent({
-        contents: [{ role: 'user', parts: [{ text: prompt }] }],
+      const response = await this.genAI.models.generateContent({
+        model: "gemini-2.0-flash",
```

ESLint

[error] Replace `"gemini-2.0-flash"` with `'gemini-2.0-flash'`

(prettier/prettier)